### DISCUSSION TITLE
GlueStackUI performance issues

### DISCUSSION THREAD

**Discussion by atanaskanchev** [gluestack/gluestack-ui#879] [Q&A]
Question for all that use the library: do you find the performance of the components adequate? According to [this benchmark](https://github.com/efstathiosntonas/react-native-style-libraries-benchmark) GlueStack is 1112.38% slower at rendering an array of 1000 empty views than native stylesheets. Is this your experience in real life usage?

**Comment by itishermann** [comment]
i have some little inconveniences but it's still better than native base though

**Comment by Viraj-10** [comment]
Hey @atanaskanchev, visit [this benchmark](https://github.com/gluestack/gluestack-ui-benchmarks) for more clarifications. Yes It is slower than Native Stylesheet but it comes with features. Also we are actively working to improve performance of `gluestack`. 
you can have a look at this example [repo](https://github.com/gluestack/ui-examples) which is built using `gluestack-ui`.

**Comment by likeSo** [comment]
The only reason that i use gluestack: It is faster then Native Base. That is enough
